python_ta/checkers/forbidden_io_function_checker.py

```diff
@@ -5,6 +5,8 @@
 from pylint.checkers import BaseChecker
 from pylint.checkers.utils import only_required_for_messages
 
+from python_ta.checkers.global_variables_checker import is_in_main
```

Comment:
It's not common for one checker to import another. Instead, please move the `is_in_main` function into the `utils` module, and then import it into both checkers.

Reply:
Makes sense! Moved to the utils module.